### PR TITLE
Fixed regex issue in safari

### PR DESCRIPTION
**Pull request by AshenI99**

### CODE REVIEW

File: packages/components/src/Header.tsx [hunk 1]
```diff
@@ -197,7 +197,7 @@ export const Header = ({
                             display: 'inline-block',
                         }}
                     >
-                        {titleArray[1] && titleArray[0]}
+                        {titleArray[1] && titleArray[0]}/
```
Comment (achrafismyname) [inline packages/components/src/Header.tsx:200]: There is an issue with this if the title is just one word. `/` shouldn't stay if the title is just one word.
You can see it here
https://60d9b2ac2cf86a00396f9b0f-lgjetjlwos.chromatic.com/?path=/story/components-header--disconnected

**Review by achrafismyname — COMMENTED** [review]
Thank you

File: packages/components/src/Header.tsx (hunk 1 above)
Comment (AshenI99) [reply packages/components/src/Header.tsx]: Could you check it now?

Comment (achrafismyname) [reply packages/components/src/Header.tsx]: It's working, good solution!

Comment (AshenI99) [reply packages/components/src/Header.tsx]: Thanks 👍

**Pull request merged**